### PR TITLE
Is1838/fogbugz areas

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

In #1838, the use wants to distinguish cases in fogbugz for OSPARC, S4L and TIS solutions. This PR adds in statics settings to configure different fogbugz url per product.

The values in ``.env-devel`` sets for ever product different areas in the same fb project

### extra
- cleanup fixtures in pytest_simcore plugin
- upgrades all ``aio-pika`` to latest everywhere. Noticed that in sidecar random error (log not found), there is an TypeError in the library. 
image

## Related issue number
#1838

## How to test

```cmd
make devenv
cd servcies/web/server
make install-dev
pytest tests/unit/isolated
```

## Checklist

- [x] Did you change any service's API? Then make sure to bundle document and upgrade version (``make openapi-specs``, ``git commit ...`` and then ``make version-*``)
- [x] Unit tests for the changes exist
- [x] Runs in the swarm
- [x] Documentation reflects the changes
- [x] New module? Add your github username to [.github/CODEOWNERS](.github/CODEOWNERS)

### CODE REVIEW

File: services/web/client/source/class/osparc/desktop/NavigationBar.js [hunk 1]
```diff
@@ -394,13 +394,16 @@ qx.Class.define("osparc.desktop.NavigationBar", {
...
+          const fbNewIssueUrl = osparc.utils.issue.Fogbugz.getNewIssueUrl(statics);
+          if (fbNewIssueUrl) {
+            window.open(fbNewIssueUrl);
+            issueConfirmationWindow.close();
+          }
         }
       }, this);
       const loginBtn = new qx.ui.toolbar.Button(this.tr("Log in in Fogbugz"), "@FontAwesome5Solid/external-link-alt/12");
-      loginBtn.addListener("execute", () => window.open("https://z43.manuscript.com/login"), this);
+      loginBtn.addListener("execute", () => window.open("https://z43.fogbugz.com/login"), this);
```
Comment (odeimaiz) [inline services/web/client/source/class/osparc/desktop/NavigationBar.js:406]:
```suggestion
      loginBtn.addListener("execute", () => {
        const statics = this.__serverStatics;
        if (statics && statics.fogbugzLoginUrl) {
          window.open(statics.fogbugzLoginUrl);
        }
      }, this);
```

**Review by odeimaiz — APPROVED** [review]
lgtm

File: .env-devel [hunk 2]
```diff
@@ -60,8 +60,10 @@ WEBSERVER_RESOURCES_DELETION_TIMEOUT_SECONDS=900
 WEBSERVER_GARBAGE_COLLECTION_INTERVAL_SECONDS=30
 WEBSERVER_MANUAL_MAIN_URL=http://docs.osparc.io/
 WEBSERVER_MANUAL_EXTRA_URL=https://itisfoundation.github.io/osparc-manual-z43/
-WEBSERVER_FOGBUGZ_URL=htpps://z43.fogbugz.com
-WEBSERVER_FOGBUGZ_PROJECT_ID=45
+WEBSERVER_FOGBUGZ_LOGIN_URL=https://z43.fogbugz.com/login
+WEBSERVER_FOGBUGZ_NEWCASE_URL=https://z43.fogbugz.com/f/cases/new?command=new&pg=pgEditBug&ixProject=45&ixArea=449
+WEBSERVER_S4L_FOGBUGZ_NEWCASE_URL=https://z43.fogbugz.com/f/cases/new?command=new&pg=pgEditBug&ixProject=45&ixArea=458
+WEBSERVER_TIS_FOGBUGZ_NEWCASE_URL=https://z43.fogbugz.com/f/cases/new?command=new&pg=pgEditBug&ixProject=45&ixArea=459
```
Comment (sanderegg) [inline .env-devel:66]: minor: should we add these things in the DB?

**Review by sanderegg — APPROVED** [review]

File: .env-devel (hunk 2 above)
Comment (pcrespov) [reply .env-devel:66]: these are static settings for the front-end

**Pull request merged**